apluslms_roman/cli.py

```diff
@@ -210,6 +212,11 @@ def add_cli_actions(parser):
         help=_("build the course (default action)"))
     build.add_argument('course', nargs='?',
         help=_("location of the course definition (default: current working dir)"))
+    build.add_argument('-s', '--steps', nargs='+',
```

Comment:
I'm not yet sure if `nargs='+'` or `action='append'`. Though, because we have positional argument `course`, then command `roman build -s foo bar some_path` becomes obscure and would require format `roman build some_path -s foo bar`, which would be relatively ok. I'm not sure eactly how ArgumentParser handles this case internally. Biggest concern is that `roman build -s foo bar` is parsed, so `{'steps': 'foo', 'course': 'bar'}`.

Thus, safer option would be the `action='append'` and would then require format `roman build -s foo -s bar some_path` witch is overly verbose and is reason why I was looking for option `roman build -s foo,bar some_path` or if I want to be verbose `roman build --steps=foo,bar some_path`.

Reply:
I removed the `course` parameter in current milestone0.3, so the `nargs='+'` is now the better alternative. Still, it would be nice to be able to write `--steps=foo,bar` too.

Reply:
I'm making the assumption that step names don't have commas in them